services/app-api/src/postgres/contractAndRates/prismaSharedContractRateHelpers.ts

```diff
@@ -262,9 +266,23 @@ function ratesRevisionsToDomainModel(
         domainRevisions.push(domainRevision)
     }
 
-    domainRevisions.sort(
-        (a, b) => a.rate.createdAt.getTime() - b.rate.createdAt.getTime()
-    )
+    return domainRevisions
+}
+
+function unsortedRatesRevisionsToDomainModel(
```

Comment:
This seems misnamed now- it looks like we are sorting before we feed into this function and what this function is doing is just changing those sorted rate revisions from prisma to domain model types